Helper/Config.php

```diff
@@ -515,7 +515,7 @@ public function getAutoCaptureOpenInvoice(int $storeId): bool
 
     public function getSupportMailAddress(int $storeId): ?string
     {
-        return $this->getConfigData('adyen_support_email_address', self::XML_ADYEN_SUPPORT_PREFIX, $storeId);
+        return $this->getConfigData('adyen_support_email_address', self::XML_ADYEN_ABSTRACT_PREFIX, $storeId);
```

Comment:
Can you please remove `XML_ADYEN_SUPPORT_PREFIX` from the constants also?

Reply:
Should be resolved.